opm/autodiff/FlowMainEbos.hpp

```diff
@@ -187,16 +187,15 @@ namespace Opm
             if (output_cout_) {
                 const int lineLen = 70;
                 const std::string version = moduleVersionName();
-                const std::string banner = "This is flow (version "+version+")";
+                const std::string banner = "This is flow "+version;
                 const int bannerPreLen = (lineLen - 2 - banner.size())/2;
                 const int bannerPostLen = bannerPreLen + (lineLen - 2 - banner.size())%2;
                 std::cout << "**********************************************************************\n";
                 std::cout << "*                                                                    *\n";
                 std::cout << "*" << std::string(bannerPreLen, ' ') << banner << std::string(bannerPostLen, ' ') << "*\n";
                 std::cout << "*                                                                    *\n";
-                std::cout << "* Flow is a simulator for fully implicit three-phase black-oil flow, *\n";
-                std::cout << "*            and is part of OPM. For more information see:           *\n";
-                std::cout << "*                       http://opm-project.org                       *\n";
+                std::cout << "*   Flow is a fully-implicit simulator for geologic applications.    *\n";
```

Comment:
I think this is too unspecific. The old message was better...! But I agree that it was not precise, since the solvent and polymer capabilities were added in the same program. What about:
```
Flow is a simulator for fully implicit three-phase black-oil flow,
       including solvent and polymer capabilities.
```
The two-phase model is a special limited case and we need not advertise it separately in the banner.

Reply:
IMO, this is as good. if you want it changed I'll open a new PR. one question before doing this: what is the intended purpose of this output? making the user aware of what they use (i.e., marketing) or is it mainly a debugging tool to quickly see what was used from the logfiles? a mix of both?

Reply:
Both. I find it useful to see what version I have (also on the command line) when using a VM for example (to answer which version got installed). Also, it is useful marketing to tell people, "Hey, we do polymer and solvent as well!".

Reply:
ok, thanks for the clarification and the suggestion. I hope #1301 does the trick...